scRNA-seq/02-tag-based_pre-processing_scRNA-seq.md

```diff
@@ -106,7 +106,7 @@ later on, including files with information on the UMIs and cell barcodes.
 Copy and paste this in your command line to run Alevin quantification.
 ```
 salmon alevin -l ISR \
-  -i data/human_index \
+  -i index/Mus_musculus/short_index \
```

Comment:
Also consistent with what we have here: https://github.com/AlexsLemonade/training-modules/blob/master/RNA-seq/01-download_qc_quant.md#transcriptome-index--i 👍